frontends/web/package.json

```diff
@@ -7,8 +7,8 @@
     "build": "yarn run gen-ts-css && preact build --no-sw --no-esm --no-inline-css --no-prerender --no-prerenderUrls --no-json --verbose",
     "dev": "yarn run gen-ts-css && preact watch --no-sw --no-esm --no-prerender --no-prerenderUrls --no-json --refresh",
     "prelint": "yarn run gen-ts-css",
-    "lint": "eslint --max-warnings 0 --cache --format unix --ignore-path ../../.gitignore 'src/**/*.{js,jsx,ts,tsx}'",
-    "fix": "eslint --fix --no-cache --ignore-path ../../.gitignore 'src/**/*.{js,jsx,ts,tsx}'",
```

Comment:
@BrianCraig but don't you want eslint to actually ignore directories like node_modules and build/... when running the lint? shouldn't this be:

```
eslint ... --ignore-path .gitignore src/...
```

Reply:
Cool, this should be updated, but it isn't showing here, check Files changed tab here on the PR